client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -540,6 +543,51 @@ function recordOrderInWPcomGA4( cart, orderId, wpcomJetpackCartInfo ) {
 	debug( 'recordOrderInWPcomGA4: Record WPcom Purchase in GA4' );
 }
 
+/**
+ * Sends a purchase event to Google Tag Manager for eligible Woo Express upgrades.
+ *
+ * @param {import('@automattic/shopping-cart').ResponseCart} cart
+ * @param {string} orderId
+ * @param {import('@automattic/data-stores/src').SiteDetails} selectedSiteData
```

Comment:
Same comment here about importing from a package's public exports rather than reaching inside the package.

```suggestion
 * @param {import('@automattic/data-stores').SiteDetails} selectedSiteData
```